src/components/storage/InstallationScenario.jsx

```diff
@@ -125,13 +129,42 @@ const checkMountPointMapping = ({ mountPointConstraints, selectedDisks, usablePa
     return availability;
 };
 
-// TODO implement
-const checkHomeReuse = ({ selectedDisks }) => {
+const checkHomeReuse = ({ devices, originalExistingSystems, selectedDisks }) => {
```

Comment:
No need to pass as property the devices and originalExistingSystems. 

We have custom hooks for that - to make the code cleaner.

const originalExistingSystems = useOriginalExistingSystems();
const devices = useOriginalDevices();

Reply:
I was thinking we don't use any hooks / side effects in the checks so I wanted to stick to it also in this check. But it seems these two hooks (using _Original_) don't have any side effects so in this case it does not matter.
Also devices is already passed to checkConfiguredStorage so I wanted to be consistent.

Reply:
>  I will use the `useOriginalExistingSystems` in the check

Hm, rather not:
```
#   134:37  error  React Hook "useOriginalExistingSystems" is called in function "checkHomeReuse" that is neither a React function component nor a custom React Hook function. React component names must start with an uppercase letter. React Hook names must start with the word "use"  react-hooks/rules-of-hooks
```